pkg/kubectl/cmd/delete.go

```diff
@@ -218,6 +218,8 @@ func (o *DeleteOptions) Validate(cmd *cobra.Command) error {
 			o.WaitForDeletion = true
 			o.GracePeriod = 1
 		}
+	} else if o.ForceDeletion {
```

Comment:
There's a function `printDeletion` that prints "resource foo deleted" after deletion operation is sent successfully. 

Would you change the operation to "force deleted" when `o.GracePeriod == 0`? So that `kubectl delete pod foo --grace-period=0 --force` prints `pod "foo" force deleted`.

Reply:
@janetkuo  Modified.
cc @soltysh